packages/date-picker/src/DatePicker/DatePicker.tsx

```diff
@@ -80,11 +91,31 @@ export interface DatePickerProps
    * e.g. disabledDaysOfWeek={[DayOfWeek.Mon, DayOfWeek.Tue]}
    */
   disabledDaysOfWeek?: DayOfWeek[]
+  /**
+   * Callback when a date is attempted to be selected.
+   * ValidationResponse object will be returned.
+   * @returns {ValidationResponse}
```

Comment:
Technically wrong since `ValidationResponse` is the param provided to the consumer, and not the returned value of the callback (which is `void`).